.github/workflows/ci.yml

```diff
@@ -620,11 +620,8 @@ jobs:
             bash -euo pipefail -c \
             "./build_tools/cmake/build_riscv.sh && tests/riscv32/smoke.sh"
 
-  # TODO(#10391): Update the benchmark dependency with the new benchmark
-  # framework. The step only requires the compiled benchmark modules, not the
-  # result from the benchmark.
   linux_riscv:
```

Comment:
This will eventually be extended to all the cross-compile platforms, but I need some boilerplate script to be in place

Reply:
See the follow up PR at https://github.com/iree-org/iree/pull/10746